gems/quilt_rails/README.md

```diff
@@ -89,6 +89,8 @@ Will run the application, starting up both servers and compiling assets.
 
 An application can also be setup manually using the following steps.
 
+[Generate Rails boilerplate](https://github.com/Shopify/quilt/blob/master/gems/quilt_rails/README.md#generate-rails-boilerplate)
```

Comment:
This link works, but I think you can also just change it to `[Generate Rails boilerplate](#generate-rails-boilerplate)`